.vscode/settings.json

```diff
@@ -2,5 +2,9 @@
     "python.autoComplete.extraPaths": [
         "/home/alexemdesgagne/Workspaces/ros_sonia_ws/devel/lib/python2.7/dist-packages",
         "/opt/ros/melodic/lib/python2.7/dist-packages"
+    ],
+    "python.analysis.extraPaths": [
+        "/home/alexemdesgagne/Workspaces/ros_sonia_ws/devel/lib/python2.7/dist-packages",

```

Comment:
I see that you wrote the path to your personnal directory, is it something mandatory ?